src/geometry/GeoParams.cc

```diff
@@ -31,11 +38,41 @@ namespace celeritas
  */
 GeoParams::GeoParams(const char* gdml_filename)
 {
-    CELER_LOG(info) << "Loading from GDML at " << gdml_filename;
     {
         detail::ScopedTimeAndRedirect time_and_output_;
+
+#ifndef CELERITAS_USE_ROOT
+        CELER_LOG(info) << "VecGeom parsing: Loading from GDML at "
+                        << gdml_filename;
         constexpr bool                validate_xml_schema = false;
         vgdml::Frontend::Load(gdml_filename, validate_xml_schema);
+#else
+
+#    ifdef VECGEOM_ROOT
+        // use Root available from VecGeom
+        CELER_LOG(info) << "RootGeoManager parsing: Loading from GDML at "
+                        << gdml_filename;
+        vecgeom::RootGeoManager::Instance().set_verbose(1);
+        vecgeom::RootGeoManager::Instance().LoadRootGeometry(gdml_filename);
+
+#    else
+        // use Root available from VecGeom
+        CELER_LOG(info) << "Root TGeoManager parsing: Loading from GDML at "
+                        << gdml_filename;
+        TGeoManager::Import(gdml_filename);
+
+        vecgeom::GeoManager::Instance().Clear();
+        TGeoNode const* const world_root = ::gGeoManager->GetTopNode();
+        // Convert() will recursively convert daughters
+        Stopwatch timer;
+        timer.Start();
+        auto fWorld = Convert(world_root);
+        timer.Stop();
+        std::cout << "*** Conversion of ROOT -> VecGeom finished ("
+                  << timer.Elapsed() << " s) ***\n";
```

Comment:
This code won't compile (`Stopwatch` doesn't have accessors like that) and has extra debug code, the comments are wrong -- clearly this `#else` branch hasn't been tested. In what circumstances will it actually be called? Maybe there should be two choices:
```c++
#if  CELERITAS_USE_ROOT && VECGEOM_ROOT
```
?

Reply:
Indeed, the build fails with `vecgeom+cuda-root` because then the `Convert()` function in L69 does not exist.
So this can only be used with Root.  I have removed the whole block containing the Stopwatch timer and the call to `Convert()`.